src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx

```diff
@@ -79,25 +75,23 @@ For accounts on New Relic One pricing, some high-level billing information is di
 
   <Collapser
     id="user-count"
-    title="User count"
+    title="Full user count billing details"
   >
-    For accounts on [New Relic One pricing](/docs/accounts/accounts-billing/new-relic-one-pricing-users/pricing-billing), one billing factor is the monthly count of [full users](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-type), which you can see tracked [in the UI](#billing-usage-ui).
-
-    Here's an overview of how the monthly full user count is calculated:
+For accounts on [New Relic One pricing](/docs/accounts/accounts-billing/new-relic-one-pricing-users/pricing-billing), one billing factor is the monthly count of provisioned [full users](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-type). For example, if you're on the Pro pricing tier and your organization has 100 full users present for the month of January, you'd be billed for 100 full users for that month. A full user counts as a billable user the moment they're provisioned (added to a New Relic organization), whether or not that user has logged into or used New Relic yet. 
```

Comment:
"your organization has 100 full users **present for** the month of January" --> "your organization has 100 full users **in** the month of January"